packages/statemanager/src/stateManager.ts

```diff
@@ -748,6 +749,91 @@ export class DefaultStateManager implements EVMStateManagerInterface {
     return returnValue
   }
 
+  /**
+   * Create a StateManager and initialize this with proof(s) gotten previously from getProof
+   * This generates a (partial) StateManager where one can retrieve all items from the proof
+   * @param proof Either a proof retrieved from `getProof`, or an array of those proofs
+   * @param safe Wether or not to verify that the roots of the proof items match the reported roots
+   * @param opts DefaultStateManagerOpts
```

Comment:
Shouldn't we rename this and expand the doc to something like: 

```suggestion
   * @param verifyRoot verify that all proof root nodes match statemanager's stateroot - should be set to `false` when constructing a state manager where the underlying trie has proof nodes from different state roots
```